quickstart/llmd-infra-installer.sh

```diff
@@ -35,6 +37,8 @@ Options:
   -f, --values-file PATH           Path to Helm values.yaml file (default: values.yaml)
   -u, --uninstall                  Uninstall the llm-d components from the current cluster
   -d, --debug                      Add debug mode to the helm install
+  -i, --skip-infra)                 SKIP_INFRA=true; shift;;
```

Comment:
its confusing to call it `infra` in this context since this script is literally called `llmd-infra-installer`

Is there a better, more specific flag we can use?